R/get_draws_df.R

```diff
@@ -66,7 +95,17 @@ get_draws_df.data.frame <- function(x,
                                     count_data,
                                     stan_data_list,
                                     fit_obj,
+                                    included_variables =
+                                      c(
+                                        "predicted counts",
+                                        "predicted wastewater",
+                                        "global R(t)",
+                                        "subpopulation R(t)"
+                                      ),
                                     ...) {
+  # Make sure that the specified variables match what is expected
+  included_variables <- rlang::arg_match(included_variables)
```

Comment:
I will remove this, I was trying to write something that would allow the user to specify which output variables they wanted but I weirdly could not get the filter to work, and really this should be in a separate PR. Will remove!